captum/attr/_utils/attribution.py

```diff
@@ -111,6 +118,22 @@ def compute_convergence_delta(self, attributions, *args):
             "Deriving sub-class should implement" " compute_convergence_delta method"
         )
 
+    @classproperty
+    def display_name(self):
```

Comment:
`display_name ` sounds UI specific, can we name it more generic  ? Can we actually, look into PyTorch and use similar convention ? For the module they have `get_name()` https://pytorch.org/docs/stable/_modules/torch/nn/modules/module.html#Module

Does it have to be a class property can we use static method like this ?

https://github.com/automl/Auto-PyTorch/blob/master/autoPyTorch/core/autonet_classes/autonet_image_classification.py#L20?